src/components/atoms/ScrollToTopButton/index.test.tsx

```diff
@@ -0,0 +1,28 @@
+import { fireEvent, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { testRenderer } from "../../../setupTests";
+import ScrollToTopButton from "./index";
+
+describe("<ScrollToTopButtom />", () => {
+  it("doesn't display ScrollButton when the page is render", async () => {
+    testRenderer(<ScrollToTopButton />);
+
+    expect(
+      screen.queryByTitle(/Ikona przewijania strony do początku/i)
+    ).not.toBeVisible();
+  });
+
+  // I think we need to test this case in the component when the button will be implemented
```

Comment:
Do you mean used in ProblemDetailedPage? :)

Reply:
Yes :)

Reply:
I think we can test it here, but we need to render `Navbar`, `ProblemsPage` and `Footer`:
`testRenderer(
      <>
        <Navbar />
        <ProblemsPage />
        <Footer />
      </>
    )`

Reply:
Exactly ;) IMO this is the same case as it was with input. We don't want to render other components than the exact one we  would like test.

Reply:
Yes, let's wait until then and we will move it to ProblemDetailedPage test. 👌